index.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+        <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <title>Cristiano Ronaldo vs Lionel Messi 2020</title>
+        <link rel="shortcut icon" type="png/image" href="/assets/images/logos/favicon.ico">
+    </head>
+
+    <body>
+        <header>
+            <nav class="firstnav">
+                <i class="fa fa-bars fa-sm"></i>
+                <img src="https://www.freeiconspng.com/uploads/youtube-logo-png-transparent-image-5.png" alt="">
+            </nav>
+           
+            <form class="search">
+                <input type="text" placeholder="Search" value=""/>
+                <button><i class="fa fa-search"></i></button>
+            </form>
+
+            <i class="fa fa-upload fa-lg" aria-hidden="true"></i>
+            
+            <i class="fa fa-th"></i>
+
+            <i class="fa fa-user fa-lg"></i> 
+
```

Comment:
1. There should be three images on the right side of the search button. Currently, you are using font awesome icons. [pathright screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png)

Reply:
Three icons on the right side of the search button have been replaced with images.
Thanks. Awaiting approval.